package.json

```diff
@@ -23,15 +22,19 @@
     "jsdom": "^11.11.0",
     "jsdom-global": "^3.0.2",
     "mocha": "^5.2.0",
+    "react": "^16.3.0",
```

Comment:
Had to add `react` as a `devDependency` as well, because `yarn` has no way of installing peer dependencies!!! I find that so weird... https://github.com/yarnpkg/yarn/issues/1503 it's a big mess 🤦‍♂️ 

I checked what big projects do about this, and `react-redux` puts `react` and `redux` in both `peerDependencies` and `devDependencies`, so I just opted to do the same thing:

https://github.com/reduxjs/react-redux/blob/aba2452c8012336daaca666a3fda0f565a8f0cbc/package.json#L88